src/pymortests/conftest.py

```diff
@@ -14,9 +14,9 @@
 settings.register_profile("ci_large", max_examples=400, **_common_settings)
 settings.register_profile("ci_pr", max_examples=100, **_common_settings)
 settings.register_profile("ci", max_examples=25, **_common_settings)
-settings.register_profile("dev", max_examples=10, **_common_settings)
+settings.register_profile("dev", max_examples=20, **_common_settings)  # 10 is not enough for vectorarrays.py
```

Comment:
There's [a mechanism](https://docs.pymor.org/2021-1-0/environment.html) to select which profile you want to run. Are you aware of it?

Reply:
commit is dropped by now